src/pages/AppProfile.vue

```diff
@@ -126,37 +126,20 @@
                   v-model="organisationDetails.size"
                 />
               </div>
-              <div
-                class="flex column mobile-remove"
-                style="width: 280px; visibility: hidden"
-              ></div>
-            </div>
-            <div class="flex flex-wrap" style="justify-content: space-between">
               <div class="flex column details">
                 <span class="body-2">Country</span>
                 <span
-                  class="sub-heading-3 overflow-ellipsis"
-                  :title="organisationDetails.country"
-                  v-if="!editOrganisationDetails"
+                    class="sub-heading-3 overflow-ellipsis"
```

Comment:
Please check the indentation here and in the hidden div below.

Reply:
Just realized what's going on, we should add a `.editorconfig` file.